### PR TITLE
Add local filesystem capability to util/gcs.

### PR DESCRIPTION
**Pull request by michelle192837**
Have checked that local files work with Merger and Updater (though not Summarizer as-is; that will need an update to how the storage works, will tackle in a follow-up PR).

### CODE REVIEW

File: util/gcs/gcs.go [hunk 1]
```diff
@@ -75,8 +79,8 @@ func (g *Path) SetURL(u *url.URL) error {
 	switch {
 	case u == nil:
 		return errors.New("nil url")
-	case u.Scheme != "gs":
-		return fmt.Errorf("must use a gs:// url: %s", u)
+	case u.Scheme != "gs" && u.Scheme != "":
```
Comment (chases2) [inline util/gcs/gcs.go:82]: nit: consider tolerating `u.Scheme == "file"`, ex. "file:///tmp/config"

**Review by chases2 — APPROVED** [review]
The polymorphism here looks good to me. May need to do a BUILD.bazel update?
/lint

File: util/gcs/client.go [hunk 2]
```diff
@@ -76,60 +75,54 @@ type ConditionalClient interface {
 	If(read, write *storage.Conditions) ConditionalClient
 }
 
-// NewClient returns a GCSUploadClient for the storage.Client.
-func NewClient(client *storage.Client) ConditionalClient {
-	return realGCSClient{client, nil, nil}
+type GCSClient struct {
```
[the remark this answers is not in the document]
Comment (fejta) [reply util/gcs/client.go:78]: No need to export this structure.

**Comment by fejta** [comment]
Please git add the new files!

File: util/gcs/gcs.go (hunk 1 above)
Comment (michelle192837) [reply util/gcs/gcs.go]: Done I think!

**Review by chases2 — APPROVED** [review]

**Pull request merged**